GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
+## Role
+  * Group of permissions
+  * Can be attached to a group
+## Rule
+## Tag
```

Comment:
Tag - A named collection of Relays. Tags are used to determine which relays bundles are available on. Additionally, tags may be used within rules to signify that matching command invocations should be executed on a relay associated with that tag. (Future Feature)